lsp/client-extension/src/extension.ts

```diff
@@ -1,61 +1,54 @@
-/* --------------------------------------------------------------------------------------------
- * Copyright (c) Microsoft Corporation. All rights reserved.
- * Licensed under the MIT License. See License.txt in the project root for license information.
- * ------------------------------------------------------------------------------------------ */
-
 import { PathLike } from 'fs';
-import * as path from 'path';
 import { workspace, ExtensionContext } from 'vscode';
+import { lookupInPath } from './toolchain';
 
 import {
 	LanguageClient,
 	LanguageClientOptions,
 	ServerOptions,
-	TransportKind
 } from 'vscode-languageclient/node';
 
 let client: LanguageClient;
 
-export function activate(context: ExtensionContext) {
-	// The server is implemented in node
-	const debugServerModule = context.asAbsolutePath(
-		path.join('..','..','target', 'debug', 'nls')
-	);
-
-	const serverConfigutration = workspace.getConfiguration("nls.server")
-	const enableDebug: Boolean = serverConfigutration.get("debugLog")
-	const traceFile: PathLike = serverConfigutration.get("trace")
-	const serverModule: string = serverConfigutration.get("path")
+export async function activate(context: ExtensionContext) {
+	const serverConfigutration = workspace.getConfiguration("nls.server");
```

Comment:
I know this wasn't introduced by this PR, but taking this opportunity...

```suggestion
	const serverConfiguration = workspace.getConfiguration("nls.server");
```